tests/std/tests/P0355R7_calendars_and_time_zones_dates/test.cpp

```diff
@@ -976,6 +977,7 @@ constexpr void year_month_day_last_test() {
                     assert((y / mdl).ok());
                 } else {
                     assert(!(y / mdl).ok());
+                    assert((y / mdl).day().ok() || (y / mdl).day() == day{42});

```

Comment:
Similarly, whatever the bogus value ends up being, I think this should be commented as an implementation-specific assumption.

(Our tests are not yet intended to be run against any other STL like libc++, but that's a possibility in the future, and it just makes them easier to understand years down the road, as implementations and Standards change.)

Reply:
Adding `"see GH-1647" to this one caused it to wrap so I just omitted it here. I thought the other would suffice since it's hopefully obvious it's the same implementation-specific assumption being made.

Reply:
I guess it could also just be `// GH-1647`?

Reply:
Either is good, I'm happy with the `// implementation-specific assumption` that you picked.